pkg/image/controller/trigger/image_trigger_controller_test.go

```diff
@@ -643,7 +648,6 @@ func scenario_1_buildConfig_imageSource_defaultImageTag() *buildv1.BuildConfig {
 						Namespace: "other"},
 					LastTriggeredImageID: "image/result:2",
 				}},
-				{ImageChange: &buildv1.ImageChangeTrigger{From: &corev1.ObjectReference{Kind: "DockerImage", Name: "mysql", Namespace: "other"}}},
```

Comment:
yeah this is prevented in the apiserver ... see https://github.com/openshift/openshift-apiserver/blob/4ec57eb921bad25a4e72151137f81cccfd63762e/pkg/build/apis/build/validation/validation.go#L642